src/patternfly/components/Alert/sr.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="pf-u-sr-only"> 
```

Comment:
This file should live in `/utilities/Accessibility`. I'm not sure of the scope of this issue, but if it's to convert the "accessibility" code to handlebars, we should have either 1) a hbs file for each `sr-only`, `visible`, and `hidden`, or 2) a single `accessibility.hbs` file that takes `sr-only`, `visible`, and `hidden` as arguments (similar to the `display` utility's hbs file). I would probably do option 2.

And I don't see any changes to the alignment utility code - should those be introduced with this issue/PR?

Reply:
ya I think that's a good idea. I'd say option two makes the most sense. @christiemolloy if you are ok with it can you take care of that in this PR.

Reply:
that sounds good, ill add it back into the scope of the issue/pr